bindings/node/test/autoEncrypter.test.js

```diff
@@ -183,6 +184,13 @@ describe('AutoEncrypter', function() {
   });
 
   describe('autoSpawn', function() {
+    beforeEach(function() {
+      if (process.env.NODE_SKIP_LIVE_TESTS) {
+        this.test.skip();
+        sandbox.restore();
```

Comment:
is this call to `restore` needed? Does is a sandbox ever set up in this case?

Reply:
It might not be needed anymore, I will check locally.